databases/week1/database-week1.sql

```diff
@@ -0,0 +1,39 @@
+-- 1 Find out how many tasks are in the task table
+SELECT COUNT(`title`) FROM `task`; 
+-- SELECT `title` FROM `task` ;
```

Comment:
Try to avoid committing empty or commented lines  for a good practice
This applies to lines 3,7,11 & so on.